### DISCUSSION TITLE
After updating from 9.2.2 to 9.4.4 type-checking is not performed anymore in runtime

### DISCUSSION THREAD

**Discussion by alexeymarkov** [vercel/next.js#14216] [Help] [answered]
I have a custom server configured.
In 9.2.2 when I start a project from vscode using "node server.js" and then change a ts/tsx file it goes compiled and type checked.
In 9.4.4 type-checking is not executed although I see tslint errors in vscode.

In 9.2.2 I see in the output
_bundled successfully, waiting for typecheck results..._
in 9.4.4 I do not see the line

Do I need to adjust something in next/babel configuration?

**Comment by rafaelalmeidatk** [comment, marked as answer]
Next.js doesn't type-check the code during development anymore since it slows down the build (https://github.com/vercel/next.js/issues/12735#issuecomment-626869676). The editor will warn you about any type errors and you can manually check the types for the entire project running `tsc`.

Type-checking is still performed during `next build`.
